Content.Shared/_CM14/Tackle/TackleSystem.cs

```diff
@@ -44,7 +44,7 @@ private void OnDisarmed(Entity<TackleableComponent> target, ref CMDisarmEvent ar
         if (recently.Current < tackle.Threshold)
         {
             var targetName = Identity.Name(target, EntityManager, args.User);
-            _popup.PopupClient($"You try to tackle {targetName}", args.User, args.User);
+            _popup.PopupClient(Loc.GetString("cm-tackle-try-self", ("target", targetName)), args.User, args.User);
```

Comment:
The targetName line above can be removed since it doesn't need to be used for localization anymore.
```suggestion
            _popup.PopupClient(Loc.GetString("cm-tackle-try-self", ("target", target.Owner)), args.User, args.User);
```